aspnetcore/blazor/forms-validation.md

```diff
@@ -199,14 +200,63 @@ The `ValidationMessage` and `ValidationSummary` components support arbitrary att
 
 ::: moniker range=">= aspnetcore-3.1"
 
-**Microsoft.AspNetCore.Blazor.DataAnnotations.Validation package**
+### Blazor data annotations validation package
 
 The [Microsoft.AspNetCore.Blazor.DataAnnotations.Validation](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.DataAnnotations.Validation) is a package that fills validation experience gaps using the `DataAnnotationsValidator` component. The package is currently *experimental*, and we plan to add these scenarios into the ASP.NET Core framework in a future release.
```

Comment:
Try to avoid using the word "we"

Reply:
Saying that it's currently experimental suggests that one day it won't be. I think it's fine to kill that clause.

WRT "try to avoid," I'm aware of the guidance. "We" is used >230 times on the repo in over 100 files. "We recommend" is the most common. "We plan" appears far less frequently, but that might be because it's temporary. I don't think "we plan" is a serious problem in relation to other problems here.